openlibrary/utils/solr.py

```diff
@@ -91,15 +92,14 @@ def select(self, query, fields=None, facets=None,
         if len(payload) < 500:
             url = url + "?" + payload
             logger.info("solr request: %s", url)
-            data = urllib.request.urlopen(url, timeout=10).read()
+            jsonData = requests.get(url, timeout=10).json()
         else:
             logger.info("solr request: %s ...", url)
             if not isinstance(payload, bytes):
                 payload = payload.encode("utf-8")
-            request = urllib.request.Request(url, payload, {"Content-Type": "application/x-www-form-urlencoded; charset=UTF-8"})
-            data = urllib.request.urlopen(request, timeout=10).read()
+            jsonData = requests.post(url, data=payload, headers={"Content-Type": "application/x-www-form-urlencoded; charset=UTF-8"}, timeout=10).json()
```

Comment:
```suggestion
            headers={"Content-Type": "application/x-www-form-urlencoded; charset=UTF-8"}
            jsonData = requests.post(url, data=payload, headers=headers, timeout=10).json()
```
Readability: We should not make new code that is > 88 characters per line max.